lib/src/layout_element.dart

```diff
@@ -244,10 +244,59 @@ TableStyleElement parseTableDefinitionElement(dom.Element element,
   }
 }
 
-LayoutElement parseLayoutElement(dom.Element element,
+class DetailsContentElement extends LayoutElement {
+  List<dom.Element> title;
+
+  DetailsContentElement({
+    String name,
+    List<StyledElement> children,
+    dom.Element node,
+    this.title,
+  }) : super(name: name, node: node, children: children);
+
+  @override
+  Widget toWidget(RenderContext context) {
+    return ExpansionTile(
+        title: title.first.localName == "summary" && children != null ? StyledText(
+          textSpan: TextSpan(
+            style: style.generateTextStyle(),
+            children: [children
+                .map((tree) => context.parser.parseTree(context, tree))
+                .toList().first] ??
+                [],
+          ),
+          style: style,
+        ) : Text("Details"),
+        children: [
+          StyledText(
+            textSpan: TextSpan(
+              style: style.generateTextStyle(),
+              children: children
+                  .map((tree) => context.parser.parseTree(context, tree))
+                  .toList() ??
+                  [],
+            ),
+            style: style,
+          ),
+        ]
+    );
+  }
+}
+
+LayoutElement parseLayoutElement(
+    dom.Element element,
     List<StyledElement> children,
 ) {
   switch (element.localName) {
+    case "details":
+      return DetailsContentElement(
+          node: element,
+          name: element.localName,
+          //todo
```

Comment:
Can this be merged with this TODO? Seems like removing the tag might be important, but I haven't worked on the issue, so you'd know better

Reply:
It probably can, it's a pretty minor issue. Basically details and summary work like this

<details><summary>Summary shows only in the title of the dropdown</summary>

Content goes here
</details>

but in the current implementation the package renders it like this:

<details><summary>Summary shows only in the title of the dropdown</summary>
Summary shows only in the title of the dropdown

Content goes here
</details>

The title gets duplicated in the content section. I just didn't know how I could stop that from getting rendered in the content because its technically a child of <details>. I might be overcomplicating it, do you have any ideas?

Reply:
@ryan-berger gave it another shot, turns out I was just overcomplicating the implementation. This is now ready to merge with no todos.

Reply:
Let me take a look! Sorry I didn't have time to respond the other night again, but I saw this. I had a solution to your problem, but let me take a look at your updated code because it sounds like your issue was resolved

Reply:
Sure no problem, I messaged on my most recent PR (#530) that it should be merged first, then this can be merged so I can apply the changes to this code as well.